packages/edit-site/src/components/routes/use-title.js

```diff
@@ -16,7 +16,7 @@ import { unlock } from '../../lock-unlock';
 
 const { useLocation } = unlock( routerPrivateApis );
 
-export default function useTitle( title ) {
+export default function useTitle( title, accessibleTitle ) {
```

Comment:
we could avoid to pass a second parameter

Reply:
Done.